From a7a92a06a47322dee45664db5c7bfa68b2b95edb Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Wed, 23 Sep 2020 07:03:02 +0000
Subject: [PATCH] Remove obsolete setting regarding the Standard Output

The Standard output type "syslog" is obsolete, causing a warning since systemd
version 246 [1].

Please consider using "journal" or "journal+console"

[1] https://github.com/systemd/systemd/blob/master/NEWS#L202

Upstream-Status: Submitted [https://github.com/lxc/lxc/pull/3541]

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
 config/init/systemd/lxc@.service.in         | 2 --
 1 files changed, 2 deletions(-)

diff --git a/config/init/systemd/lxc@.service.in b/config/init/systemd/lxc@.service.in
index a2aa2211f..21ce64aef 100644
--- a/config/init/systemd/lxc@.service.in
+++ b/config/init/systemd/lxc@.service.in
@@ -14,8 +14,6 @@ ExecStop=@BINDIR@/lxc-stop -n %i
 # Environment=BOOTUP=serial
 # Environment=CONSOLETYPE=serial
 Delegate=yes
-StandardOutput=syslog
-StandardError=syslog
 
 [Install]
 WantedBy=multi-user.target
-- 
2.26.2

